### PR TITLE
Testcase to validate signed/unsigned char,short as normalized float

### PR DESCRIPTION
**Pull request by ansurya**
Added new testcase to validate char,uchar,short,ushort texture as normalized float.

### CODE REVIEW

File: tests/src/texture/hipNormalizedFloatValueTex.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+/* HIT_START
+ * BUILD: %t %s ../test_common.cpp EXCLUDE_HIP_PLATFORM nvcc
```
Comment (mangupta) [inline tests/src/texture/hipNormalizedFloatValueTex.cpp]: Is there any specific reason why we are excluding this on nvidia? Ideally this test should work there as well.

File: tests/src/texture/hipNormalizedFloatValueTex.cpp [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ * TEST: %t
+ * HIT_END
+ */
+
+#define CHECK(cmd) \
```
Comment (gargrahul) [inline tests/src/texture/hipNormalizedFloatValueTex.cpp]: There is HIP_CHECK already defined in test_common.h , which is included below.

File: tests/src/texture/hipNormalizedFloatValueTex.cpp [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    desc.Height = 1;
+    desc.Format = texFormat;
+    desc.NumChannels = 1;
+    CHECK(hipTexRefSetAddress2D(texRef, &desc, (hipDeviceptr_t)dData, sizeof(T)*SIZE));
+    
+    bool testResult = true;
+    float *dOutputData = NULL;
+    CHECK(hipMalloc((void **) &dOutputData, sizeof(float)*SIZE));
+ 
+    hipLaunchKernelGGL(HIP_KERNEL_NAME(normalizedValTextureTest), dim3(1,1,1), dim3(SIZE,1,1), 0, 0, SIZE, dOutputData);
```
Comment (gargrahul) [inline tests/src/texture/hipNormalizedFloatValueTex.cpp]: Do we really need HIP_KERNEL_NAME here?

**Comment by gargrahul** [comment]
@mangupta @ansurya  There is another bigger issue with this test. hipTexRefXXXX APIs are equivalent to CUDA driver APIs whereas this sample has used these as runtime APIs. This is why I think we should try to run tests on NVCC path as well.

**Pull request merged**